py/visdom/static/css/style.css

```diff
@@ -129,6 +129,13 @@ button {
   overflow: hidden;
 }
 
+.widgets {
+  position: absolute;
+  bottom: 0;
```

Comment:
Seems that by being absolutely positioned, there is the possibility that the widgets panel can be overlaid on top of the content. Could this cause any issues where the underlying content is not "interactive-able"?

Instead of making `widgets` a first class citizen and exposing it through the `Pane` API what are the trade-offs of having the higher abstraction (in this case `ImagePane`) making it part of the content prop instead? Are there any benefits in having widgets be elevated as a specific additional prop?

Reply:
This debate is exactly the reason I didn't want to merge these changes without having a discussion with someone haha. One advantage I imagined down the line is as people start wanting more interactive components, they'd have the ability to wire them directly into the widgets of relevant plots/panes. 

Of course the position absolute on the bottom at the moment can cover up content, and I couldn't come up with a different solution to be able to append additional content greater than the specified pane height.

Reply:
Agreed on the idea of extensibility, though I'm still tossing around the execution in my mind. I think for a v0 this is fine, though I'd like to revisit this at some point.

As an aside, I'm noticing out-of-place text (repeated caption plus an errant `<`) underneath the image. Unrelated to this PR:
![visdom-image-move](https://user-images.githubusercontent.com/425059/55588375-95b5a400-56fb-11e9-9f57-b6d6901472fe.gif)

Reply:
Well *that* explains where the old captions went 😂